toqito/state_ops/tests/test_schmidt_decomposition.py

```diff
@@ -257,8 +96,8 @@ def test_schmidt_decomp_random_state():
         ],
         axis=0,
     )
-    bool_mat = np.isclose(rho, reconstructed)
-    np.testing.assert_equal(np.all(bool_mat), True)
+    assert np.isclose(rho, reconstructed).all()
+
 
 
 def test_schmidt_decomp_random_operator():
```

Comment:
It appears L132-136 is covered by this test but it still shows up as uncovered in the `pytest` summary.